Action updating visual test reference snapshots fails

As seen in a few recent PRs the action updating snapshots fails:
- https://github.com/jupyterlab/jupyterlab/pull/15612 https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637
- https://github.com/jupyterlab/jupyterlab/pull/15623 https://github.com/jupyterlab/jupyterlab/actions/runs/7472322849
- https://github.com/jupyterlab/jupyterlab/issues/14766#issuecomment-1883075772 https://github.com/jupyterlab/jupyterlab/actions/runs/7461896604

It seems to have last worked on
- 29th Dec 2023 for 4.0.x backports (https://github.com/jupyterlab/jupyterlab/pull/15578#issuecomment-1872073508, https://github.com/jupyterlab/jupyterlab/actions/runs/7356925384/job/20027751189)
- 27th Dec 2023 for `main` https://github.com/jupyterlab/jupyterlab/actions/runs/7338404549/job/19980875086

and was no longer working by 6th Jan 2024 (https://github.com/jupyterlab/jupyterlab/pull/15612#issuecomment-1879643925)

```
Run jupyterlab/maintainer-tools/.github/actions/update-snapshots@v1
  with:
    github_token: ***
    server_url: http-get://localhost:8888/lab
    test_folder: galata
    artifact_name: updated-galata-snapshots
    report_name: update-galata-report
    npm_client: yarn
    browser: chromium
    continue_on_error: yes
    dry_run: no
    git_username: github-actions[bot]
    git_email: github-actions[bot]@users.noreply.github.com
    start_server_script: start
    update_script: test:update
  env:
    PYTHON_VERSION: 3.11
    NODE_VERSION: 18.x
    CACHE_PREFIX: Linux-Update Playwright Snapshots-update-galata-snapshots
    DEPENDENCY_TYPE: standard
    pythonLocation: /opt/hostedtoolcache/Python/3.11.7/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.7/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.7/x64
    Python[2](https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637/job/20423240394#step:8:2)_ROOT_DIR: /opt/hostedtoolcache/Python/[3](https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637/job/20423240394#step:8:3).11.7/x6[4](https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637/job/20423240394#step:8:4)
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.7/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.7/x64/lib
Run (yarn run start 1>/tmp/snapshots-update-server.log 2>&1) &
  (yarn run start 1>/tmp/snapshots-update-server.log 2>&1) &
  shell: /usr/bin/bash -l {0}
  env:
    PYTHON_VERSION: 3.11
    NODE_VERSION: 18.x
    CACHE_PREFIX: Linux-Update Playwright Snapshots-update-galata-snapshots
   ``` DEPENDENCY_TYPE: standard
    pythonLocation: /opt/hostedtoolcache/Python/3.11.7/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.7/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.7/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.7/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.7/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.7/x64/lib
Run yarn install
/home/runner/work/_temp/[5](https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637/job/20423240394#step:8:5)413518f-ee12-4f[6](https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637/job/20423240394#step:8:6)3-b[7](https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637/job/20423240394#step:8:7)df-[8](https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637/job/20423240394#step:8:8)5cea0c4[9](https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637/job/20423240394#step:8:9)52f.sh: line 1: yarn: command not found
/home/runner/work/_temp/5413518f-ee12-4f63-b7df-85cea0c4952f.sh: line 2: yarn: command not found
Error: Process completed with exit code 127.
Run echo "::group::Server log"
  echo "::group::Server log"
  cat /tmp/snapshots-update-server.log
  echo "::endgroup::"
  shell: /usr/bin/bash -l {0}
  env:
    PYTHON_VERSION: 3.[11](https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637/job/20423240394#step:8:11)
    NODE_VERSION: 18.x
    CACHE_PREFIX: Linux-Update Playwright Snapshots-update-galata-snapshots
    DEPENDENCY_TYPE: standard
    pythonLocation: /opt/hostedtoolcache/Python/3.11.7/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.7/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.7/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.7/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.7/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.7/x64/lib
Server log
  /home/runner/work/_temp/5f502a24-5f43-4dd6-8417-8[16](https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637/job/20423240394#step:8:16)[30](https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637/job/20423240394#step:8:31)c3f[60](https://github.com/jupyterlab/jupyterlab/actions/runs/7501823637/job/20423240394#step:8:63)7b.sh: line 1: yarn: command not found
```

--- a/.github/workflows/galata-update.yml
+++ b/.github/workflows/galata-update.yml
@@ -51,6 +51,7 @@ jobs:
           test_folder: galata
           artifact_name: updated-galata-snapshots
           report_name: update-galata-report
+          npm_client: jlpm
 
       - name: Comment back on the PR
         run: |

